src/subscription.ts

```diff
@@ -306,34 +325,43 @@ export class Subscription extends Subscriber {
    *   const apiResponse = data[1];
    * });
    */
-  createSnapshot(name, gaxOpts, callback) {
+  createSnapshot(name: string, callback?: CreateSnapshotCallback): void;
```

Comment:
`callback` should be required param here (no `?`);

Reply:
changes done!